src/main/conversions/c2_15_0_2017081601.clj

```diff
@@ -0,0 +1,28 @@
+(ns facepalm.c2-15-0-2017081601
+    (:use [korma.core :exclude [update]]
+          [kameleon.sql-reader :only [exec-sql-statement]]
+          [kameleon.uuids]))
+
+(def ^:private version
+  "The destination database version"
+  "2.15.0:20170816.01")
+
+(defn- add-ols-ontology-term-value-type
+  "Updates the value_types view."
```

Comment:
This is hyper-nitpicky but `value_types` isn't a view!

Reply:
Nope, good catch 😆